README.md

```diff
@@ -1,3 +1,17 @@
 # About
 
 This repository contains the execution of the EF standard execution layer tests.
+
+# Setup
```

Comment:
you should add a markdown linter to your IDE!

Reply:
<img width="435" alt="image" src="https://github.com/kkrt-labs/ef-tests/assets/18620296/846398dd-de7a-4131-a2f7-5995d2672fb8">